lib/compat/wordpress-6.3/rest-api.php

```diff
@@ -85,15 +85,16 @@ function add_modified_wp_template_schema() {
 }
 add_filter( 'rest_api_init', 'add_modified_wp_template_schema' );
 
-/**
- * Registers the block patterns REST API routes.
- */
-function gutenberg_register_rest_block_patterns() {
-	$block_patterns = new Gutenberg_REST_Block_Patterns_Controller_6_3();
-	$block_patterns->register_routes();
+if ( ! gutenberg_is_experiment_enabled( 'gutenberg-auto-inserting-blocks' ) ) {
```

Comment:
Is it worth adding a comment that when the experiment is enabled, the block patterns controller is registered in `experimental/rest-api.php`? It took a second to realise that it's now happening in one of two places.

Reply:
Yeah, good point -- I'll do that now 👍

Reply:
edffdc1136